CHANGES.rst

```diff
@@ -29,6 +29,7 @@ Internal Changes
 - Use webpack to build static assets
 - Add React+Redux for new frontend modules
 - Enable modern ES201x features
+- Fix incorrect order of session blocks inside timetable (:issue:`2999`)
```

Comment:
it's a bugfix, not an internal change (add the "Bugfixes" section after "Improvements" like we did for other versions)